src/bank00.asm

```diff
@@ -107,11 +108,15 @@ LCDCInterruptHandler:
     pop  AF                                            ;; 00:00a8 $f1
     reti
 
+lotmsInit:
+    DBG_MSG_LABEL introDebugMsg
+    jp FullReset
```

Comment:
As far as I can tell there's no reason to prefer `jp` over `jr` if targets are in range. Since you are touching this anyway you can change them.